### PR TITLE
please sleep

### PR DESCRIPTION
**Pull request by Niki4tap**
don't mind hook function for now, it's for future, or you can wipe it out, idk, refactored some code, simplified and united status function exists now, fixed like 4 bugs, need sleep now

### CODE REVIEW

File: src/editor.hpp [hunk 1]
```diff
@@ -29,8 +28,9 @@ class Editor {
 	void deleteCharL();
     void deleteCharR();
 	
-	void addSimpleStatus(const std::string& name, const std::string& status_message);
-	void enableSimpleStatus(const std::string& name);
+	void printColoredStatus(const std::string& message, const int& colorPair);
```
Comment (xyl1t) [inline src/editor.hpp]: `const int&`? just do `int`

Comment (Niki4tap) [reply src/editor.hpp:31]: breaks colors, don't change it

Comment (Niki4tap) [reply src/editor.hpp]: Ah yeah, you can, fixed this in 3a712f55a6ee9142680da600eaaef86a4389eccf

**Review by xyl1t — COMMENTED** [review]
nais

**Pull request merged**